navigator/frontend/src/applets/templatecontracts/columns.tsx

```diff
@@ -1,26 +1,21 @@
 // Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
 // SPDX-License-Identifier: Apache-2.0
 
-import {
-  ChoicesButton,
-  ContractColumn,
-} from '@da/ui-core';
-import { DamlLfDataType, DamlLFFieldWithType } from '@da/ui-core/lib/api/DamlLfType';
-import { DamlLfValueRecord } from '@da/ui-core/lib/api/DamlLfValue';
+import {ChoicesButton, ContractColumn} from '@da/ui-core';
+import {DamlLFFieldWithType, DamlLfRecord} from '@da/ui-core/lib/api/DamlLfType';
```

Comment:
Merge this line with the `import {DamlLfValueRecord} from '@da/ui-core/lib/api/DamlLfValue';` line below

Reply:
fixed